bundles/org.openhab.binding.cp750/README.md

```diff
@@ -0,0 +1,48 @@
+# CP750 Binding
+
+This binding is intended to connect to a _Dolby Digital Cinema Processor CP750_.
+
+It uses ASCII commands send over a TCP connection on port 61408. Please note that the CP750 only accepts up to 20 simultaneous connection and will discard the oldest connection if a 21st connection is established. So be sure to grateful shutdown each connection using the client's close() method, or it's AutoCloseable functionality.
+
+This binding wraps the CP750 Java library from https://github.com/Cybso/cp750-java/.
+
+This project is NOT affiliated with, funded, or in any way associated with Dolby Laboratories, Inc.
+
+Currently, only _Fader_, _Mute_ and _Input Mode_ channels are implemented.
+
+## Supported Things
+
+- `cp750` 
+
+## Thing Configuration
+
+Normally, only the hostname or IP address must be configured.
+
+### `sample` Thing Configuration
+
+| Name              | Type    | Description                                       | Default | Required | Advanced |
+|-------------------|---------|---------------------------------------------------|---------|----------|----------|
+| hostname          | text    | Hostname or IP address of the device              | N/A     | yes      | no       |
+| port              | integer | TCP port if different from 61408                  | 61408   | no       | no       |
+| refreshInterval   | integer | Interval the device is polled in sec.             | 5       | no       | no       |
+| reconnectInterval | integer | Interval a new connection is tried after IO error | 10      | no       | no       |
+
+## Channels
+
+_Here you should provide information about available channel types, what their meaning is and how they can be used._
+
+_Note that it is planned to generate some part of this based on the XML files within ```src/main/resources/OH-INF/thing``` of your binding._
+
+| Channel | Type   | Read/Write | Description                                                                                                     |
+|---------|--------|------------|-----------------------------------------------------------------------------------------------------------------|
+| version | Switch | R          | Version as reported by the device                                                                               |
```

Comment:
See other comment about version in thing-typ.xml, if you need to have it as channel, i suggest to change it to os-version as that is more specific and possibly more future proof.